db/seeds.rb

```diff
@@ -44,6 +46,12 @@
 player_1a = Player.create!(
   name: "Jackson Smylie", rating: 2006, division: 1, crosstables_id: 20032, tournament: tourney_1
 )
+
+jackson_photo = URI.open("https://www.cross-tables.com/pix/IMG_1335.jpg")
```

Comment:
For the seed, can we just reuse an asset instead of downloading and storing a new one each time?

Reply:
Absolutely - this was just to test that cloudinary actually works